lib/src/protocol/kawa_h1/parser.rs

```diff
@@ -64,22 +65,36 @@ impl Method {
     }
 }
 
-impl fmt::Display for Method {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+impl AsRef<str> for Method {
```

Comment:
We should keep a `Display` implementation that you the `AsRef<str>`

Reply:
the implementation of Dispaly is still present, only five lines below :-)